release/k8s_tests/run_gcs_ft_on_k8s.py

```
@@ -401,6 +403,10 @@ def main():
             for p in procs:
                 p.kill()
             print("==== Cleanup done ===")
+
+        if exception:
+            raise exception
```

mmm what is happening here?

---

We reraise the exception after cleanup - this test always exited successfully despite not actually working, as the exception was swallowed

---

Why don't we raise it in row 392?

---

I didn't want to modify the flow of the script (cleanup) - @sihanwang41 will be rewriting the test anyway

---

@sihanwang41 , where are we tracking the rewrite?